### PR TITLE
[v6r19] Fixes for RSS scripts and transformation system tests

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES
*tests
NEW: RSS system test

*Core
FIX: removing docs and tests directories when creating release tarballs.

ENDRELEASENOTES

### CODE REVIEW

File: ResourceStatusSystem/scripts/dirac-rss-query-db.py [hunk 1]
```diff
@@ -349,8 +349,8 @@ def add( args, switchDict ):
                                                tokenExpiration = getToken( 'expiration' )
                                              )
 
-  if 'Value' in output:
-    result['match'] = int( output['Value'] )
+  if 'Value' in output and output['Value']:
```
Comment (chaen) [inline ResourceStatusSystem/scripts/dirac-rss-query-db.py]: ```if output.get('Value')``` ?

File: ResourceStatusSystem/scripts/dirac-rss-query-dtcache.py [hunk 2]
```diff
@@ -300,7 +300,8 @@ def add( switchDict ):
   if not output['OK']:
     return output
 
-  result['match'] = int( output['Value'] )
+  if output['Value']:
```
Comment (chaen) [inline ResourceStatusSystem/scripts/dirac-rss-query-dtcache.py]: Here you do not do the same test as in dirac-rss-query-db purposedly ?

File: TransformationSystem/scripts/dirac-transformation-add-files.py [hunk 3]
```diff
@@ -13,18 +13,17 @@
...
+                                   ] ) )
 
 Script.parseCommandLine()
 
 from DIRAC.TransformationSystem.Client.TransformationClient import TransformationClient
 
 args = Script.getPositionalArgs()
-if ( len( args ) != 2 ):
+if len( args ) != 2:
   Script.showHelp()
```
Comment (chaen) [inline TransformationSystem/scripts/dirac-transformation-add-files.py]: Don't you need to exit ?

File: TransformationSystem/scripts/dirac-transformation-get-files.py [hunk 4]
```diff
@@ -12,31 +12,22 @@
...
+                                   ] ) )
 
 Script.parseCommandLine()
 
 from DIRAC.TransformationSystem.Client.TransformationClient import TransformationClient
 
 args = Script.getPositionalArgs()
-if ( len( args ) != 1 ):
+if len( args ) != 1:
   Script.showHelp()
```
Comment (chaen) [inline TransformationSystem/scripts/dirac-transformation-get-files.py]: exit here ?

File: ResourceStatusSystem/scripts/dirac-rss-query-dtcache.py (hunk 2 above)
Comment (fstagni) [inline ResourceStatusSystem/scripts/dirac-rss-query-dtcache.py]: There should be no need here.

**Comment by fstagni** [comment]
I corrected according the suggestions.

**Comment by chaen** [comment]
review ok

**Pull request merged**